apps/test-bundles/webpackUtils.js

```diff
@@ -86,53 +139,42 @@ const TopLevelEntryFileExclusions = ['index.js', 'version.js', 'index.bundle.js'
  * for components which are just re-exported.
  */
 function buildEntries(packageName, entries = {}, includeStats = true, onlyOwnComponents = false) {
```

Comment:
Fix onlyOwnComponents

Reply:
Probably best to either fix onlyOwnComponents or just remove it for now. Removing it would probably be fine since we're not currently using it.